cdk-infra/updateSearchMappings/index.ts

```diff
@@ -73,6 +73,6 @@ exports.handler = async (event: any) => {
                 break;
         }
     } catch (e) {
-        await sendCfnResponse(event, 'FAILED', e);
+        await sendCfnResponse(event, 'FAILED', e as Error);
     }
 };
```

Comment:
Why were the few files below deleted?

Reply:
This was actually a mistake in the gitignore, i've fixed this!